src/nwnsdk/postgres/postgres_client.py

```diff
@@ -62,14 +59,46 @@ def store_job_result(self, job_id: uuid4, new_logs: str, new_status: JobStatus,
             stmnt = (
                 update(Job)
                 .where(Job.job_id == job_id)
-                .values(status=new_status.value, logs=new_logs, output_esdl=output_esdl, stopped_at=datetime.now())
+                .values(status=new_status, logs=new_logs, output_esdl=output_esdl, stopped_at=datetime.now())
             )
             session.execute(stmnt)
 
+    def get_job_status(self, job_id: uuid4) -> JobStatus:
+        LOGGER.debug("Retrieving job status for job %s", job_id)
+        with session_scope() as session:
+            stmnt = select(Job).options(load_only(Job.status)).where(Job.job_id == job_id)
+            job: Job = session.scalar(stmnt)
+            session.expunge(job)
+        return job.status
+
     def get_job(self, job_id: uuid4) -> Job:
-        session: Session
         LOGGER.debug("Retrieving job data for job %s", job_id)
         with session_scope() as session:
             stmnt = select(Job).where(Job.job_id == job_id)
             job: Job = session.scalar(stmnt)
+            session.expunge(job)
```

Comment:
Explicit expunging is not necessary. We already call `Session.remove()` in the `session_scope()` which in turn calls `session.close()` which expunges all objects for us:

https://docs.sqlalchemy.org/en/20/orm/contextual.html#sqlalchemy.orm.scoped_session.remove
https://docs.sqlalchemy.org/en/20/orm/session_api.html#sqlalchemy.orm.Session.close

Reply:
It is necessary else it would remove commits (inserts/updates) before committing. Queries on the other hand require expunging. I think it would be best to create a commit_scope and a query_scope. In the commit scope we start a transaction and commit. In the query scope we expunge and do not need transactions/commits.

Reply:
Then we would need a commit_scope, query_scope and commit_and_query_scope.
This would limit flexibility if adding a query or commit later. I propose to add a 'do_expunge=False' param to session_scope function.

Reply:
Also, since not all query calls will need an expunge